### DISCUSSION TITLE
Render hook 'start of end' topbar

### DISCUSSION THREAD

**Discussion by goyne** [filamentphp/filament#10962] [Help] [answered]
Hi,

I am struggling with getting an action button in the right place via renderhooks.

In my topbar, I placed a button via the renderhook 'panels::topbar.end', and logically the button is then placed at the end, see screen dump.

How do I get this button to the left of the notifications bell? Or is that not possible? I think I'm looking for a hook like 'panels::topbar.end-left' or 'panels::topbar.end-start'.

![Scherm­afbeelding 2024-01-21 om 12 36 48](https://github.com/filamentphp/filament/assets/2893526/6c7993e2-4a18-4548-916f-6591a52ad5e4)

Thanks, Jelger

**Comment by medfrk** [comment, marked as answer]
You can try 'panels::user-menu.before' or 'panels::global-search.after'

**Reply by goyne** [reply]
global-search.after did the trick. Thanks, much appreciated!
